genie-web/src/main/resources/static/scripts/components/JobDetails.js

```diff
@@ -46,9 +51,19 @@ export default class JobDetails extends React.Component {
     const url = row._links.self.href;
     fetch(url).done(job => {
       this.setState({ job });
-      fetch(job._links.command.href)
-        .done(command => {
-          this.setState({ command });
+
+      $.when(
+        fetch(job._links.cluster.href),
+        fetch(job._links.command.href),
+        fetch(job._links.applications.href)
+      )
+        .done((clusters, commands, applications) => {
+          console.log(applications);
```

Comment:
Forgot to remove console log?

Reply:
@mprimi The console log was removed in the next commit. Maybe when we squash all the commit this won't be an issue?